src/components/StartCourseButton.svelte

```diff
@@ -1,23 +1,59 @@
 <script>
+// @ts-nocheck
+  export let course_id
+  export let slug
+  export let status = "Unsubscribed"
+
   import { mutation } from "svelte-apollo";
   import { START_COURSE } from "$lib/mutations/courses";
 
+
   const startCourse = mutation(START_COURSE);
 
-  async function handleClick() {
+  async function handleStart() {
     try {
       const result = await startCourse({
         variables: {
           user_id: "a4f9150e-4619-4b18-807d-2f273b75c12f",
-          course_id: "6477f941-f072-4f1e-95da-b1d02a81ae6f",
+          course_id,
         },
       });
-      console.log(result);
+
+      if (result.data.insert_user_courses.affected_rows === 1) {
+      console.log('Mutation successful!');
+      //  Change the URL to the Svelte page
+      window.location.href = `/courses/${slug}`;
+    } else {
+      console.log('Mutation failed');
+    }
+
+      
     } catch (error) {
       console.log(error);
       // TODO
     }
   }
+
+  function handleContinue() {}
+  function handleCompleted() {}
+
+  const buttons = {
+    Completed: {
+      text: "Course Completed",
+      style: "btn btn-success",
```

Comment:
This is neat! We will consider adding types in a subsequent PR. Great work!

Reply:
Definitely